### ISSUE TITLE
Link to similarly named locations

### ISSUE THREAD

**Issue by svenseeberg** [Integreat/app-cross_platform#433]
To prevent users from actually ending up in the wrong location, we should add a hint if mix-ups with other locations are possible.

This can be achieved by splitting up the current location name and check if any word is part of any other location. Only checks for full matches are required, substrings are probably not useful.

If the check is positive, add a visible but not disturbing note somewhere on the screen, that can be dismissed.

Coordinate design with https://issues.integreat-app.de/browse/NATIVE-26

**Comment by dkehne** [comment]
Yes. Maybe we just highlight the nearest 3 instances in front of the list. Nevertheless they should be also still visible in the "all locations list", so "move" might be are misinterpreting verb here.
